jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/handler/DelayedHandlerTest.java

```diff
@@ -213,87 +213,24 @@ public void process(Request request, Response response, Callback callback) throw
         }
     }
 
-    @Test
-    public void testQualityOfService() throws Exception
-    {
-        final int QOS = 3;
-        final int EXTRA = 2;
-
-        DelayedHandler delayedHandler = new DelayedHandler.QualityOfService(QOS);
-        _server.setHandler(delayedHandler);
-
-        AtomicInteger processing = new AtomicInteger();
-        Semaphore semaphore = new Semaphore(0);
-        CountDownLatch complete = new CountDownLatch(QOS + EXTRA);
-        delayedHandler.setHandler(new HelloHandler()
-        {
-            @Override
-            public void process(Request request, Response response, Callback callback) throws Exception
-            {
-                processing.incrementAndGet();
-                semaphore.acquire();
-                super.process(request, response, Callback.from(callback, complete::countDown));
-            }
-
-            @Override
-            public InvocationType getInvocationType()
-            {
-                return InvocationType.BLOCKING;
-            }
-        });
-        _server.start();
-
-        Socket[] socket = new Socket[QOS + EXTRA];
-        for (int i = 0; i < socket.length; i++)
-        {
-            socket[i] = new Socket("localhost", _connector.getLocalPort());
-            String request = "GET /p" + i + " HTTP/1.1\r\n" +
-                "Host: localhost\r\n" +
-                "\r\n";
-            OutputStream output = socket[i].getOutputStream();
-            output.write(request.getBytes(StandardCharsets.UTF_8));
-            output.flush();
-        }
-
-        await().atMost(5, TimeUnit.SECONDS).until(processing::get, equalTo(QOS));
-
-        for (int i = 0; i < socket.length; i++)
-        {
-            semaphore.release();
-            int count = i + 1;
-            await().atMost(5, TimeUnit.SECONDS).until(processing::get, equalTo(QOS + Math.min(EXTRA, count)));
-        }
-
-        assertTrue(complete.await(5, TimeUnit.SECONDS));
-
-        for (Socket value : socket)
-        {
-            HttpTester.Input input = HttpTester.from(value.getInputStream());
-            HttpTester.Response response = HttpTester.parseResponse(input);
-            assertNotNull(response);
-            assertEquals(HttpStatus.OK_200, response.getStatus());
-            String content = new String(response.getContentBytes(), StandardCharsets.UTF_8);
-            assertThat(content, containsString("Hello"));
-        }
-    }
```

Comment:
Because I removed the QoS varient of the Delayed handler.  IT was kind of the same thing as ThreadLimitHandler, but also not implemented completely.    We can review in #9056